usr.bin/ktrdump64/Makefile

```diff
@@ -0,0 +1,9 @@
+NEED_COMPAT=	64
+.include <bsd.compat.mk>
+
+PROG=	ktrdump64
+MAN=
+SRCS=	ktrdump.c
```

Comment:
Hm, might be nicer to put this in ktrdump itself (as is done for ldd) given it already needs to be modified to use `?=` to support compat versions?